scripts/safe_template_linter.py

```diff
@@ -146,6 +146,7 @@ def __init__(self, string):
         """
         self._string = string
         self._line_breaks = self._process_line_breaks(string)
+        self.eof_index = len(string)
```

Comment:
It could just be me, but a comment here explaining that this is an exclusive index would be helpful.